narayana-integration-bom/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.kie</groupId>
+    <artifactId>kie-user-bom-parent</artifactId>
+    <version>7.30.0-SNAPSHOT</version>
+    <relativePath>../kie-user-bom-parent/pom.xml</relativePath>
+  </parent>
+
+  <groupId>org.kie</groupId>
+  <artifactId>narayana-integration-bom</artifactId>
+  <packaging>pom</packaging>
+
+  <name>Narayana Integration BOM (Bill Of Materials)</name>
+  <description>
+    Import this BOM in your dependencyManagement if you want to include narayana in your project.
+  </description>
+
+  <!-- IMPORTANT: Do not declare any build things here! Declare them in kie-user-bom-parent. -->
+  <build />
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.jboss.jbossts.jta</groupId>
+        <artifactId>narayana-jta</artifactId>
+      </dependency>
```

Comment:
I am wondering where the dependencies declared here get their version from - afaik, the version properties are declared in kie-parent, but kie-parent is not used here (which is intended). I looked into jboss-parent pom and did not find them there either.

Reply:
@winklerm I propose to replace the parent by kie-parent in order to simplify the structure. WDYT?

Reply:
I have created a simple test POM importing kie-platform-bom in the following gist \[1] and it fails to build (mvn clean verify). Could you please double check if I missed anything or if we need to define the versions?

\[1] https://gist.github.com/winklerm/28ae89cd06b825e45d3cf3e411a2872f

Reply:
@Ginxo I think that we tried to avoid having kie-parent as a parent because it would pull in lots of undesired dependencyManagement. Not sure how hard that requirement was, @mareknovotny ?

Reply:
@winklerm nice catch!

How about doing the opposite: defining the version here and import this BOM to kie-parent (if we need to make sure the kie-parent still contains everything)?

Reply:
yes, that is correct for end user BOMs